src/main/groovy/wooga/gradle/unity/UnityPluginConventions.groovy

```diff
@@ -73,7 +74,19 @@ class UnityPluginConventions implements PlatformUtilsImpl {
     /**
      * The path to the Unity Editor executable
      */
-    static final PropertyLookup unityPath = new PropertyLookup(["UNITY_UNITY_PATH", "UNITY_PATH"], "unity.unityPath", { getPlatformUnityPath().absolutePath })
+    static final PropertyLookup unityPath = new PropertyLookup(["UNITY_UNITY_PATH", "UNITY_PATH"], "unity.unityPath", { getPlatformUnityPath().absolutePath })    /**
+     /**
+     * The path to the Unity Editor root folder (Unity or Unity.app in the default installation).
+     */
+    static final PropertyLookup unityRoot = new PropertyLookup(["UNITY_UNITY_ROOT", "UNITY_ROOT"], "unity.unityRoot", null)
+    /*
+     * The path to the Unity Editor root folder (Unity or Unity.app in the default installation).
+     */
+    static final PropertyLookup unityDotnet = new PropertyLookup(["UNITY_UNITY_DOTNET", "UNITY_DOTNET"], "unity.unityDotnet", null)
```

Comment:
I think the name can be ambiguous; an user could also think it refers to the dotnet version. 
What about unityDotnetPath, or just dotnetPath? 
Since these properties will already by in the extension we don't need to prefix 'unity' to them again if we can help it.